Apromore-Core-Components/Apromore-Portal/src/main/java/org/apromore/portal/dialogController/renderer/SummaryItemRenderer.java

```diff
@@ -268,13 +268,15 @@ protected Listcell renderNA() {
 
     protected Listcell renderOwner(final SummaryType summaryType) {
         Boolean isMakePublic = summaryType.isMakePublic();
-        String label;
+        String owner;
```

Comment:
Can we make this 
owner = summaryType.getOwner();

Then have only 1 if statement